packages/cx-layout/src/components/Header/token.ts

```diff
@@ -13,21 +13,48 @@
  */
 
 import { withNodeKey } from '@bodiless/core';
-
 import { cxElement } from '@bodiless/cx-elements';
-import { as } from '@bodiless/fclasses';
+import {
+  flowHoc, replaceWith, Span, withProps
+} from '@bodiless/fclasses';
 import { asHeaderToken } from './HeaderClean';
 import { cxLogo } from '../Logo';
+import { cxMenuToggler } from '../MenuToggler';
+import { cxDesktopSearch, cxSearchToggler } from '../Search';
 
 /**
- * Token that defines a basic CanvasX header.
+ * Token that defines a basic header.
  */
 const Base = asHeaderToken({
-  // Analytics: {
-  //   Search: as(withSearchDataLayer),
-  // },
   Components: {
+    MenuToggler: cxMenuToggler.Default,
+    SearchToggler: cxSearchToggler.Default,
     Logo: cxLogo.Default,
+    DesktopSearch: cxDesktopSearch.Default,
+    // @todo replace Menu placeholder
```

Comment:
This todo applies to search, menu toggler and lanuage switcher also -- these are just placeholders.  So for now i'd craate a placeholders directory with all of these in it, including the menu placeholder

Reply:
actually - i see you actually created stanalond comopnents for these.  That was out of scope, but is a god jump on search and menu, so leave as is.